figures/scripts/fig5-panels-gsva-umap.R

```diff
@@ -260,28 +261,49 @@ gsva_heatmap <- Heatmap(
   na_col = na_color$hex_codes,
   show_column_names = FALSE,
   cluster_columns = FALSE,
-  row_names_gp = grid::gpar(fontsize = 5),
+  row_names_gp = grid::gpar(fontsize = 5.8),
   top_annotation = column_heatmap_annotation,
-  heatmap_legend_param = list(direction = "horizontal")
+  heatmap_legend_param = list(direction = "vertical")
 )
 
-pdf(gsva_pdf, width = 5.75, height = 5)
-draw(gsva_heatmap, heatmap_legend_side = "bottom")
+pdf(gsva_pdf, width = 7, height = 4)
+draw(gsva_heatmap, heatmap_legend_side = "right")
 dev.off()
 
 
 
 #### broad_histology legend ----------------------------------------------------
 
 # UMAP legend
-display_group_legend <- Legend(
-  labels = names(annotation_colors_bh),
-  legend_gp = gpar(fill = annotation_colors_bh)
+# note that legend text cannot be resized in `Legend()`, only the title text
+# Legend() is not great at text wrapping so we're using a hack
+
+histology_order <- names(annotation_colors_bh)
+
+display_group_legend_plot <- tibble::as_tibble(annotation_colors_bh, 
+                  rownames = "histology") %>%
+  mutate(histology = factor(histology, 
+                            levels = names(annotation_colors_bh))) %>%
+  ggplot() + 
+  aes(x = annotation_colors_bh, y = annotation_colors_bh, color = histology) + 
+  geom_point(size = 3.75) + 
```

Comment:
These seem a touch large to me.

Reply:
Dropped in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1751/commits/15ec9447cc020cc0a368226e6b69a29abacfa3a5